packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -559,11 +560,13 @@ class Editor implements KetcherEditor {
     }
 
     this.selection(null);
+
     if (this._tool instanceof toolsMap.paste) {
       this.event.change.dispatch();
-      return;
```

Comment:
are you sure that it will not broke anything?
return; no longer needed here?

Reply:
You are right. Changing that fixes the issue, but causes other ege cases. I need to do this differently.